Ontologies.Mappings/test/WillowMappingValidationTests.cs

```diff
@@ -17,11 +18,16 @@ public WillowMappingValidationTests(ITestOutputHelper output)
         }
 
         [Theory]
-        [InlineData("Mappings.v1.Willow.mapped_v1_dtdlv2_Willow.json")]
-        public void ValidateMappedDtmisAreValidFormat(string resourcePath)
+        [InlineData("https://mapped.com/ontologies/mapping/Mapped2Willow/latest.json", LoaderType.Http)]
+        [InlineData("..\\..\\..\\..\\..\\data\\mapped_v1_dtdlv2_Willow.json", LoaderType.File)]
+        public void ValidateMappedDtmisAreValidFormat(string resourcePath, LoaderType loaderType)
```

Comment:
@connorjcantrell here is a sample of how we can use a local file and the deployed production file. Just need set the value for the local file consistently, without embedding them as resources in the project.

Reply:
Ok, our GH workflow runs the unit tests after the Ontology Mapper has generated the mapping files. The local file will exist consistently as long as the Ontology Mapper runs successfully.

Reply:
K. What is the correct path to these files? Are they checked in to the repo when done?

Reply:
I'm not sure what you mean by command line arguments. The tests need to work in both dev local environments and in PR builds. There's no opportunity to pass arguments in AFAIK.

Why can't we have the devs push their test files to a staging location whenever they generate new files, and run the tests? The staging location can be set for everyone consistently in the tests. If we are worried that there are multiple changes by different people going on to the mappings at the same time without coordinating... that's going to bite us at some point, isn't it?

Reply:
- Not sure of C#, but other languages normally support using different configurations for the same test through env vars or different config files, etc.
- It'd be more logical if we push it to the staging area only if it passes unit test, reducing the burden to host a broken file there.
- If the staging location is same for all the different PRs, depending on the timing, a test could be using a wrong file pushed by a different PR, though the chance is not too high. That's why I was suggesting to do the integration test only at the main branch. If you don't care about such scenario, we can just do it in feature branches too.

Reply:
> If we are worried that there are multiple changes by different people going on to the mappings at the same time without coordinating... that's going to bite us at some point, isn't it?

Not sure what your point is in this comment. What are you trying to argue here? Maybe I missed something.